app/views/users/_modal.html.erb

```diff
@@ -23,7 +23,7 @@
       <div class="modal-body"
            id="generic-users-modal-content">
         <%= render partial: "users/generic_users",
-                   locals: { generic_users: generic_users } %>
+                   locals: { generic_users_count: generic_users_count } %>
```

Comment:
Why don't we need the "@" here? `@generic_users_count`

Reply:
This is because the `users/_modal.html.erb` partial gets the `generic_users_count` variable passed from `users/index.html.erb`. Of course, we could use the instance variable `@generic_users_count` here and would not need to pass it down via `users/index.html.erb`, but it is considered bad practice to use instance variables in partials. Instead, it is recommended to pass variables to partials. See e.g. [here](https://andycroll.com/ruby/dont-use-instance-variables-in-partials/) or [here](https://stackoverflow.com/questions/2503838/rails-should-partials-be-aware-of-instance-variables).

Reply:
Ah ok, good to know 👍